### ISSUE TITLE
Employment page: updates required

### ISSUE THREAD

**Issue by leehawkinsssw** [SSWConsulting/SSW.Website#260]
The Employment page at https://www.ssw.com.au/ssw/Employment/default.aspx needs a few updates:

1. In "Work with the decision makers", missing apostrophe in "Managements" (should read "Management's")
2. In "SSW Getaway/Retreat", replace "guys" with something more inclusive, such as "folks" or "people"
3. Under "Permanent Roles", the Newcastle (and China) locations are not mentioned
4. The requirement to "Live within 40 mins drive from SSW" seems unnecessary now

**Comment by wicksipedia** [comment]
Will be done once #348 is complete

**Comment by wicksipedia** [comment]
> 1. In "Work with the decision makers", missing apostrophe in "Managements" (should read "Management's")
> 2. In "SSW Getaway/Retreat", replace "guys" with something more inclusive, such as "folks" or "people"
> 3. Under "Permanent Roles", the Newcastle (and China) locations are not mentioned
The requirement to "Live within 40 mins drive from SSW" seems unnecessary now

Already done - updated page https://www.ssw.com.au/employment already fixes these issues

> 4. The requirement to "Live within 40 mins drive from SSW" seems unnecessary now

This isn't strictly a content issue so dropping
